<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="CreateNewFile"/>
    <addaction name="OpenFile"/>
    <addaction name="SaveFile"/>
    <addaction name="SaveFileAs"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <addaction name="Search"/>
    <addaction name="Replace"/>
    <addaction name="Clear"/>
    <addaction name="Undo"/>
    <addaction name="Copy"/>
    <addaction name="Paste"/>
    <addaction name="Cut"/>
    <addaction name="Redo"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Текст</string>
    </property>
    <addaction name="Palette"/>
    <addaction name="FontAndSize"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Таблица</string>
    </property>
    <addaction name="Table"/>
    <addaction name="AddRow"/>
    <addaction name="AddColumn"/>
    <addaction name="DeleteRow"/>
    <addaction name="DeleteColumn"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="CreateNewFile"/>
   <addaction name="OpenFile"/>
   <addaction name="SaveFile"/>
   <addaction name="SaveFileAs"/>
   <addaction name="Table"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Search"/>
   <addaction name="Replace"/>
   <addaction name="Clear"/>
   <addaction name="Undo"/>
   <addaction name="Redo"/>
   <addaction name="Copy"/>
   <addaction name="Paste"/>
   <addaction name="Cut"/>
   <addaction name="Palette"/>
   <addaction name="FontAndSize"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="AddRow"/>
   <addaction name="DeleteRow"/>
   <addaction name="AddColumn"/>
   <addaction name="DeleteColumn"/>
  </widget>
  <action name="CreateNewFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/add-document _black.png</normaloff>:/res/images/icons/add-document _black.png</iconset>
   </property>
   <property name="text">
    <string>Создать новый файл</string>
   </property>
  </action>
  <action name="OpenFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/folder-open_black.png</normaloff>:/res/images/icons/folder-open_black.png</iconset>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="SaveFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/save_black.png</normaloff>:/res/images/icons/save_black.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="SaveFileAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/save_as_black.png</normaloff>:/res/images/icons/save_as_black.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="Search">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/search.png</normaloff>:/res/images/icons/search.png</iconset>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </action>
  <action name="Replace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/replace.png</normaloff>:/res/images/icons/replace.png</iconset>
   </property>
   <property name="text">
    <string>Замена</string>
   </property>
  </action>
  <action name="Clear">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/clear.png</normaloff>:/res/images/icons/clear.png</iconset>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="Undo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/undo.png</normaloff>:/res/images/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </action>
  <action name="Copy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/copy.png</normaloff>:/res/images/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="Paste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/paste.png</normaloff>:/res/images/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="Cut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/cut.png</normaloff>:/res/images/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
  </action>
  <action name="Redo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/un-undo.png</normaloff>:/res/images/icons/un-undo.png</iconset>
   </property>
   <property name="text">
    <string>Переделать</string>
   </property>
  </action>
  <action name="Palette">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/palette.png</normaloff>:/res/images/icons/palette.png</iconset>
   </property>
   <property name="text">
    <string>Палитра</string>
   </property>
  </action>
  <action name="FontAndSize">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/text.png</normaloff>:/res/images/icons/text.png</iconset>
   </property>
   <property name="text">
    <string>Шрифт</string>
   </property>
  </action>
  <action name="Table">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/table-list.png</normaloff>:/res/images/icons/table-list.png</iconset>
   </property>
   <property name="text">
    <string>Создать таблицу</string>
   </property>
  </action>
  <action name="AddRow">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/row.png</normaloff>:/res/images/icons/row.png</iconset>
   </property>
   <property name="text">
    <string>Добавить строку</string>
   </property>
  </action>
  <action name="AddColumn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/column.png</normaloff>:/res/images/icons/column.png</iconset>
   </property>
   <property name="text">
    <string>Добавить столбец</string>
   </property>
  </action>
  <action name="DeleteRow">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/delete row.png</normaloff>:/res/images/icons/delete row.png</iconset>
   </property>
   <property name="text">
    <string>Убрать строку</string>
   </property>
  </action>
  <action name="DeleteColumn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/images/icons/delete column.png</normaloff>:/res/images/icons/delete column.png</iconset>
   </property>
   <property name="text">
    <string>Убрать столбец</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
